CHANGELOG.md

```diff
@@ -1,110 +1,60 @@
-| Bug   | Gift options on Order Level (Commerce only)                                                                                                       | [3540][]                      |
-| Bug   | Parent product image thumbnail has been shown in mini cart instead of product itself                                                              | [3690][]                      |
-| Bug   | Long wishlist name length breaks layout                                                                                                           | [3706][]                      |
-| Bug   | Change default sort order on Search page to 'Best Match'                                                                                          | [3698][]                      |
-| Bug   | Wishlist page error when configured variant added                                                                                                 | [3703][]                      |
-| Bug   | Fix Cypress Page Builder tests                                                                                                                    | [3715][]                      |
-| Bug   | Dynamic Blocks GraphQL returns non-qualified blocks                                                                                               | [3678][], [3][]               |
-| Bug   | Remove virtual products extension                                                                                                                 | [3612][]                      |
-| Bug   | Possible wrong usage of useMemo                                                                                                                   | [3660][]                      |
-| Bug   | PWA multiple wish lists: storefront still showing create wishlist button when reaching the max number of allowed wish lists configured in Commerce| [3668][]                      |
-| Bug   | Header - Logo dimensions overlap with main content                                                                                                | [3659][]                      |
-| Bug   | Click target for long-spanning tabs is inconsistent                                                                                               | [3676][]                      |
-| Bug   | Venia sample data personalized content not updating                                                                                               | [20][]                        |
+## Highlights  
+
+The main focus of PWA Studio 12.4 is the enhanced ability to query and consume custom product attributes. While previously, this could be done, results were difficult to parse, with little ability for sorting and filtering. Now, custom product attributes are returned in their own object within the GraphQL response and can be sorted and filtered.
```

Comment:
```suggestion
The main focus of PWA Studio 12.4 is additional support for working with custom product attributes. While previously, this could be done, results were difficult to parse, with little ability for sorting and filtering. Now, custom product attributes are returned in their own object within the GraphQL response, which can be sorted and filtered.
```